pkg/process/util/util.go

```diff
@@ -136,3 +137,12 @@ func WithAllProcs(procRoot string, fn func(int) error) error {
 
 	return nil
 }
+
+// ProcessExists checks if the given pid still exists
+func ProcessExists(pid int) bool {
+	if err := syscall.Kill(pid, 0); err != nil && err == syscall.ESRCH {
```

Comment:
A comment linking to documentation on Kill -0 would be useful here. 

I think @Gui774ume recently also needed to check the liveness of processes in https://github.com/DataDog/ebpf/pull/34. Perhaps we can unify the approaches and add a standard interface for this operation in [procutil](https://github.com/DataDog/datadog-agent/tree/master/pkg/process/procutil) (our replacement for gopsutil) or another utils package.